### ISSUE TITLE
Should set `.elm` property on created vnodes

### ISSUE THREAD

**Issue by stucox** [appcues/snabbdom-virtualize#13]
I wonder if `virtualize()` should set the `.elm` property on the nodes it creates, such that:

``` js
var div = document.createElement('div');
var vnode = virtualize(div);
console.log(vnode.elm == div)  // true
```

Reasoning:

My use case for `snabbdom-virtualize` is to “pick up where I left off” with a server-rendered DOM: snabbdom’s `patch(oldVnode, newVnode)` function expects both `oldVnode` and `newVnode` to be virtual, so I was hoping to virtualize the existing DOM before `patch()`ing with the next state.

However, snabbdom needs the `.elm` reference to tie elements up properly, which `snabbdom-virtualize` doesn’t set.

Is there any reason why setting `.elm` to refer to the source node would be a bad idea?

**Comment by stucox** [comment]
Alternatively if `hooks.create()` callback passed the source node, users of this lib would be able to add this functionality themselves.

**Comment by rayd** [comment]
This should be resolved in 0.6.0 thanks to @katyo's #14
